tests/unit_tests/grid/test_face_functions.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+import pytest
+
+from resqpy.grid import Grid
+from resqpy.model import Model
+
+import resqpy.grid.face_functions as ff
+
+
+def test_face_centre(basic_regular_grid: Grid):
+    # Arrange
+    cell = None
+    axis = 0
+    zero_or_one = 0
+    expected_face_centres = np.array([[[[50.0, 25.0, 0.0], [150.0, 25.0, 0.0]], [[50.0, 75.0, 0.0], [150.0, 75.0,
```

Comment:
If you are feeling keen, you can use something like # yapf: disable and # yapf: enable (if I remember correctly), to allow manual fomatting of a few lines of code – such as this array data.